src/styles/blocks/main-title.scss

```diff
@@ -0,0 +1,218 @@
+        justify-content: flex-start;
+      }
+    }
+  }
+
+  &__exhib {
+    font-family: "IBM Plex Sans", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 16px;
+    color: #687480;
+
+    @include onTablet {
+      font-size: 14px;
+      line-height: 14px;
+      margin-right: 30px;
+    }
+
+    @include onDesktop {
+      font-size: 14px;
+      line-height: 14px;
+      margin-right: 30px;
+    }
+  }
+
+  &__date {
+    font-family: "IBM Plex Sans", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 24px;
+    color: $green;
+
+    @include onTablet {
+      line-height: 21px;
+    }
+
+    @include onDesktop {
+      line-height: 21px;
+    }
+  }
+
+  &__subtitle {
+    font-family: "Montserrat", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 28px;
+    line-height: 28px;
+    color: #0f0e08;
+    margin: 0 0 10px;
+
+    @include onTablet {
+      margin: 0 0 15px;
+    }
+
+    @include onDesktop {
+      margin: 0 0 15px;
+    }
+
+    &::after {
+      content: "";
+      width: 10px;
+      height: 10px;
+      background-color: #f14a27;
+      border: 1px solid #f14a27;
+      display: inline-block;
+
+      @include onTablet {
+        width: 15px;
+        height: 15px;
+      }
+
+      @include onDesktop {
+        width: 15px;
+        height: 15px;
+      }
+    }
+
+    @include onTablet {
+      font-size: 36px;
+      line-height: 36px;
+    }
+
+    @include onDesktop {
+      font-size: 36px;
+      line-height: 36px;
+    }
+  }
+
+  &__descr {
+    font-family: "IBM Plex Sans", sans-serif;
+    font-style: normal;
+    font-weight: 300;
+    font-size: 16px;
+    line-height: 24px;
+    color: #4e4e4e;
+    text-align: start;
+  }
+
+  &__photo {
+    width: 100%;
+    margin-top: 50px;
+    transition: transform 0.3s;
+
+    &:hover {
+      transform: scale(0.9);
+    }
+
+    @include onTablet {
+      width: 100%;
+    }
+
+    @include onDesktop {
+      width: 670px;
+      height: 520px;
+    }
+  }
+
+  &__text {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+  }
+
+  &__end {
+    margin-bottom: 180px;
+
+    @include onTablet {
+      margin-bottom: 200px;
+    }
+
+    @include onDesktop {
+      margin-bottom: 300px;
+    }
+
+    &--photo {
+      display: block;
+      width: 100%;
+    }
+
+    &--box {
+      position: absolute;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      padding: 20px 10px;
+
+      background: #fff;
+      box-shadow: 0 2px 0 rgba(26, 90, 76, 0.25);
+
+      @include onTablet {
+        padding: 45px 55px;
+      }
+
+      @include onDesktop {
+        padding: 45px 100px;
+      }
+    }
+
+    &--title {
+      font-family: "Montserrat", sans-serif;
+      font-style: normal;
+      font-weight: 400;
+      font-size: 28px;
+      line-height: 28px;
+      text-align: center;
+      color: #0f0e08;
```

Comment:
Consider using a variable for the color #0f0e08 to maintain consistency and ease of updates.